_posts/tutorial/2023-07-11-radiuss-on-aws.md

```diff
@@ -25,4 +25,4 @@ Join us again this summer for a series of tutorials in collaboration with AWS, d
 | August 22, 2023 | 9:00a.m.–11:00a.m. | ![Ascent logo]({{ site.baseurl }}/assets/img/small/llnl-ascent.png) Visualize and analyze your simulations in situ with [**Ascent**](https://github.com/alpine-dav/ascent) | [Ascent tutorial docs](https://ascent.readthedocs.io/en/latest/Tutorial.html) |
 | August 24, 2023 | 9:00a.m.–11:00a.m. | ![Axom logo]({{ site.baseurl }}/assets/img/small/llnl-axom.png) Leverage robust, flexible software components for scientific applications with [**Axom**](https://github.com/llnl/axom) | [Axom docs](https://axom.readthedocs.io/en/develop/) |
 | August 29, 2023 | 9:00a.m.–11:00a.m. | ![OSS logo]({{ site.baseurl }}/assets/img/small/llnl-opensource.png) Analyze runs of your code with [**WEAVE**](https://github.com/LLNL/weave-demos) | [WEAVE docs](https://github.com/LLNL/weave-demos) |
-| August 31, 2023 |9:00a.m.–11:00a.m. | ![Flux logo]({{ site.baseurl }}/assets/img/small/llnl-flux.png) Learn to run thousands of jobs in a workflow with [**Flux**](https://github.com/flux-framework/flux-core) | [Flux docs](https://flux-framework.readthedocs.io)<br/>[Flux tutorial](https://tutorial.flux-framework.org) |
+| August 31, 2023 |9:00a.m.–11:00a.m. | ![Flux logo]({{ site.baseurl }}/assets/img/small/llnl-flux.png) Learn to run thousands of jobs in a workflow with [**Flux**](https://github.com/flux-framework/flux-core) | [Flux docs](https://flux-framework.readthedocs.io)<br/>[Flux tutorial](https://flux-framework.readthedocs.io/en/latest/tutorials/index.html) |
```

Comment:
@hauten I think that was correct - if I remember we had a custom DNS there that exposed the tutorial. But after the fact it’s probably OK to switch over to a static page!

Reply:
@vsoch do you mean I should switch it back now or afterward? The crawler flagged the .org URL as broken.

Reply:
oh no I think this is OK (this is the old page) just that when there is a new tutorial, that link might be the right one. Post-tutorial the static page I think is OK!